src/scripts/index.js

```diff
@@ -1,10 +1,332 @@
+}
+
+/**
+ *
+ * @param {*} ev
+ */
+function allowDrop(ev) {
+  ev.preventDefault();
+}
+
+/**
+ *
+ * @param {*} ev
+ */
+function drag(ev) {
+  console.log(ev.target);
+  ev.target.setAttribute('id', 'clickedTicket');
+  ev.dataTransfer.setData('text', ev.target.id);
+}
+
+/**
+ *
+ * @param {*} ev
+ */
+function drop(ev) {
+  ev.preventDefault();
+  const data = ev.dataTransfer.getData('text');
+  const clickedTicket = document.getElementById('clickedTicket');
+  ev.target.appendChild(document.getElementById(data));
+  clickedTicket.removeAttribute('id');
+}
+
+
+/* -----------------------those 3 dots on the header--------------------------*/
+
+const popupEdit = document.getElementById('edit-name-popup');
+const closePopupEdit = document.getElementById('closeEditName');
+closePopupEdit.addEventListener('click', closePopupEditName);
+
+const saveNewName = document.getElementById('saveNewName');
+saveNewName.addEventListener('click', editColumnName);
+
+const dots = document.querySelectorAll('.edit-column');
+for (let i = 0; i < dots.length; i++) {
+  dots[i].addEventListener('click', openPopupDots);
+}
+
+const editName = document.querySelectorAll('.edit-name-column');
+for (let i = 0; i < editName.length; i++) {
+  editName[i].addEventListener('click', openPopupEditName);
+}
+
+/** Open the popup for the dots options
+ * @param {HTMLElement} event
+*/
+function openPopupDots(event) {
+  // console.log(event.target.closest('.header'));
+  const header = event.target.closest('.header');
+  const popup = header.querySelector('.popup-dots');
+  // console.log(popup);
+  popup.classList.toggle('show');
+}
+
+/**
+ *
+ * @param {*} event
+ */
+function openPopupEditName() {
+  const header = event.target.closest('.header');
+  header.querySelector('.name-column').setAttribute('id', 'columnNameChanged');
+  popupEdit.style.display = 'block';
+  event.preventDefault();
+}
+
+
+/**
+ *
+ * @param {*} event
+ */
+function editColumnName() {
+  popupEdit.style.display = 'none';
+  console.log('h');
+  event.preventDefault();
+  const name = document.getElementById('columnNewName').value;
+  const column = document.getElementById('columnNameChanged');
+  column.innerText = name;
+  column.removeAttribute('id');
+}
+
+
+/**
+ * The function for closing the popup for edit name column
+*/
+function closePopupEditName() {
+  popupEdit.style.display = 'none';
+}
+
+/* ---------------------------------------- delete column---------------------*/
+const columnToDelete = document.querySelectorAll('.delete-column');
+for (let i = 0; i < columnToDelete.length; i++) {
+  columnToDelete[i].addEventListener('click', deleteColumn);
+}
+
+/**
+ * function for deleting a column
+ * @param {HTMLElement} event
+ */
+function deleteColumn(event) {
+  const header = event.target.closest('.flex-column');
+  console.log(header);
+  const container = document.querySelector('.flex-container');
+  container.removeChild(header);
+}
```

Comment:
While it has an ok length, please split this between multiple js files.

Reply:
Fisierul css mare sa il impart in mai multe alte fisiere, nu?

Reply:
fisierul js mare :). Preferabil e sa fie mai micut. Mai ales ca e punctul de inceput.

Reply:
O sa ma uit sa vad si daca sunt propr de css nefolosite, dar asta la pasul urmator.